cmserver/cmserver.go

```diff
@@ -117,52 +128,35 @@ type CertificateProvider interface {
 // New creates new IAM server instance.
 func New(
 	cfg *config.Config, handler UpdateHandler, certProvider CertificateProvider,
-	cryptcoxontext *cryptutils.CryptoContext, insecure bool,
+	cryptocontext *cryptutils.CryptoContext, insecure bool,
 ) (server *CMServer, err error) {
 	server = &CMServer{
+		config:        cfg,
+		certProvider:  certProvider,
+		cryptocontext: cryptocontext,
+		insecureConn:  insecure,
+
+		grpcServer:        grpchelpers.NewGRPCServer(cfg.CMServerURL),
 		currentFOTAStatus: handler.GetFOTAStatus(),
 		currentSOTAStatus: handler.GetSOTAStatus(),
+		certChan:          make(<-chan *iamanager.CertInfo),
```

Comment:
Make is useless here. Rename to `certChannel` to be aligned with `stopChannel`.

Reply:
done